### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by Abhayg444**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,51 +14,45 @@ _Collaborate and work together on GitHub._
...
+  Start this step by acknowledging the previous step.
+  Define terms and link to docs.github.com.
+  Historic note: this step combines the commend, approve, and needs changes steps from the previous version.
 -->
 
-## Welcome
+## Step 3: Leave a review
 
-All great projects start with collaboration. Pull requests are the foundation of teamwork on GitHub — and pull request reviews give you the ability to work together and discuss changes specific to a pull request by commenting, requesting changes, or approving.
+_You assigned yourself! :tada:_
```
Comment (AaruzMC) [inline README.md:26]: Need more detailed explanation

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (AaruzMC) [inline index.html:51]: Give more intersting Message so the user refreshes.

**Review by AaruzMC — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -14,51 +14,30 @@ _Collaborate and work together on GitHub._
...
-3. After your new repository is created, wait about 20 seconds, then refresh the page. Follow the step-by-step instructions in the new repository's README.
+1. On the pull request, click **Files changed**.
+1. Find the `index.html` changes.
+1. Hover your cursor next to the line numbers on the left side of the page.
+1. Click the blue plus icon.
+1. After the comment form appears, click the **Add a suggestion** button. <br>
+   ![add-a-suggestion-button](https://user-images.githubusercontent.com/97056108/184449714-61e8ee51-824a-48c1-9436-2dfd67f2c070.png)
+1. Edit the suggestion.
+1. Click **Add a single comment**.
+1. Wait about 20 seconds then refresh this page (the one you're following instructions from). [GitHub Actions](https://docs.github.com/en/actions) will automatically update to the next step.
```
Comment (AaruzMC) [inline README.md:40]: Make it more elegent

**Review by AaruzMC — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (Abhayg444) [reply README.md:33]: Added

File: index.html (hunk 2 above)
Comment (AaruzMC) [inline index.html]:
```suggestion
Add more h2 message
```

**Review by AaruzMC — APPROVED** [review]

**Pull request merged**